src/udgl/layout/ContentBody.svelte

```diff
@@ -1,3 +1,11 @@
-<main class=content-content>
+<style>
+  .content-body-content {
```

Comment:
we can change this to just be `.content-body` since it's now component-scoped

Reply:
alternatively, you can ALSO just target `main` here and remove the class altogether. IMO `main` tells us semantically what the content is better than `content-body`. The rest is just style detail.

Reply:
Going with this approach would mean that you can remove a lot of classes throughout these components.